Backend/src/main/java/com/onbank/schedulers/TransferSessions.java

```diff
@@ -1,38 +1,82 @@
 package com.onbank.schedulers;
 
+import com.onbank.api.model.Transfer;
+import com.onbank.api.model.TransferState;
+import com.onbank.api.model.csv.CSVToTransfer;
+import com.onbank.api.model.csv.TransferToCSV;
+import com.onbank.api.repository.TransferRepository;
 import com.onbank.ftp.FtpConnection;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Component;
 
+import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
 
+@RequiredArgsConstructor
 @Component
 public class TransferSessions {
 
-    @Autowired
-    FtpConnection ftpConnection;
+    @Value("${onbank.server}")
+    private String server;
+    @Value("${onbank.port}")
+    private int port;
+    @Value("${onbank.user}")
+    private String user;
+    @Value("${onbank.password}")
+    private String password;
 
 
+    private final TransferRepository transferRepository;
+
     @Scheduled(cron = "${onbank.outgoing.transfer.date}")
     public void outgoingTransferMorningSession() {
         System.out.println("Session outgoing");
-        try {
-            ftpConnection.open();
-            ftpConnection.close();
+
+        List<Transfer> transfers = transferRepository.findAll().stream()
```

Comment:
To do zmiany. Nie filtrujmy w pamięci naszej aplikacji po pobraniu wszystkich rekordów, tylko już w bazie danych. Trzeba zrobić metodę w repo, która wyciągnie nam pola o wymaganych właściwościach